readthedocs/projects/tasks.py

```diff
@@ -250,6 +250,13 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
             self.version = self.get_version(version_pk)
             self.project = self.version.project
 
+            if not Project.objects.is_active(self.project):
```

Comment:
Also, this does not work because we don't have access to the DB here. We probably need to adding this to the API response.